samples/brokers/coap-core-broker/Dockerfile.coap-core

```diff
@@ -0,0 +1,21 @@
+ARG PLATFORM=amd64
+ARG CROSS_BUILD_TARGET=x86_64-unknown-linux-gnu
+FROM ${PLATFORM}/debian:buster-slim
+ARG CROSS_BUILD_TARGET
+RUN echo "Creating container based on ${PLATFORM}/debian:buster-slim"
+RUN echo "Using Rust binaries from ${CROSS_BUILD_TARGET}"
+
+# Link the container to the Akri repository
+LABEL org.opencontainers.image.source https://github.com/deislabs/akri
+
+# Copy over container legal notice
+COPY ./build/container-images-legal-notice.md .
+
+RUN apt-get update && apt-get install -y --no-install-recommends libssl-dev openssl && apt-get clean
+COPY ./target/${CROSS_BUILD_TARGET}/release/coap-core-broker /coap-core-broker
+
+# Expose port used by broker service
+EXPOSE 8083
+
+ENV RUST_LOG coap_core_broker
+CMD ["./coap-core-broker"]
```

Comment:
There are some inconsistencies on the Docker configuration side. I tried following the extensibility documentation, which didn't explain cross compilation for the broker however. So I looked at existing Docker configuration and there's a mix of things from the extensibility doc and things from the existing configs.

E.g. this file should be in `/build/containers`

Reply:
This is definitely a shortcoming of the extensibility docs.

I would suggest updating build\akri-containers.mk to include your new broker container.   Then create something like .github\workflows\build-udev-video-broker-container.yml for your broker.

Reply:
Thanks for the help, I've tried to create the CI config. Let's see if it works